### PR TITLE
enable basic auth on the mobile_auth fetch_key_records endpoint

### PR DESCRIPTION
**Pull request by dannyroberts**
This is the endpoint I saw phillip's phone hitting (http://manage.dimagi.com/default.asp?177414)

### CODE REVIEW

File: corehq/apps/mobile_auth/views.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import datetime
 from django.http import HttpResponse
 from django.views.decorators.http import require_GET
-from corehq.apps.domain.decorators import login_or_digest_ex
+from corehq.apps.domain.decorators import login_or_digest_ex, \
```
Comment (dimagimon) [inline corehq/apps/mobile_auth/views.py]: F401 'login_or_digest_ex' imported but unused

**Comment by dannyroberts** [comment]
@nickpell

**Comment by nickpell** [comment]
testing on staging

**Comment by czue** [comment]
:+1: good call

**Pull request merged**